compiler/example/datamodel/config/gns/gns.go

```diff
@@ -6,7 +6,7 @@ import (
 	cartv1 "github.com/vmware-tanzu/cartographer/pkg/apis/v1alpha1"
 
 	service_group "github.com/vmware-tanzu/graph-framework-for-microservices/compiler/example/datamodel/config/gns/service-group"
-	policypkg "github.com/vmware-tanzu/graph-framework-for-microservices/compiler/example/datamodel/config/policy"
```

Comment:
please don't rename this import, it's used for testing some kind of named import case

Reply:
addressed